src/exercise/01.html

```diff
@@ -18,20 +18,16 @@
 <!--   📜 https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append -->
 <!DOCTYPE html>
 <html lang="en">
-<head>
-   
-</head>
-<body>
+  <head> </head>
+  <body>
     <div id="root"></div>
     <script type="module">
-    var div = document.createElement('div');
-    div.textContent = "Hello World, Its Brandon";
-    div.className = "container";
-    
-    var root = document.getElementById('root');
-root.append(div);
-    
-    </script>
-</body>
+      var div = document.createElement('div');
```

Comment:
This is good! technically `const` is a more preferred variable initialization instead of `var` as the pointer to that document object will be constant... this is true also for `root` down below;

Eventually, linters will steer you toward const if we're not re-assigning the variable down the road...